README.md

```diff
@@ -18,14 +18,27 @@ providers {
 }
 ```
 
-The provider takes optional configuration to specify a `kubeconfig` file:
+The provider takes couple optional configuration parameters:
+
+* If you have a kubeconfig available on the file system you can configure the provider as:
 
 ```hcl
 provider "k8s" {
   kubeconfig = "/path/to/kubeconfig"
 }
 ```
 
+* If you content of the kubeconfig is available in a variable, you can configure the provider as:
+
+```hcl
+provider "k8s" {
+  kubeconfig_content = "${var.kubeconfig}"
+}
+```
+
+Configuration from the variable will be recorded into a temporary file and the file will be removed as
```

Comment:
Make this a warning.

"WARNING: ..."

Writing to the global tempdir would be a pretty terrifying thing to do on a shared system.